pvlib/pvsystem.py

```diff
@@ -1709,6 +1709,8 @@ def calcparams_desoto(effective_irradiance, temp_cell,
     Rs = R_s
 
     numeric_args = (effective_irradiance, temp_cell)
+    # IL: photocurrent, I0: saturation_current, Rs: resistance_series, 
+    # Rsh: resistance_shunt, nNsVth: product of thermal voltage, diode ideality factor and cells in series
```

Comment:
```suggestion
    # Rsh: resistance_shunt
```

`nNsVth` has the same name in the downstream function `singlediode`